_data/locales/gr.yml

```diff
@@ -20,3 +20,10 @@ version_messages:
   patch: "Υπάρχουν αρκετές διορθωτικές αλλαγές στο αρχικό μάθημα μετά την τελευταία ενημέρωση."
   none: "Αυτή η μετάφραση είναι πλήρως ενημερωμένη."
   error: "There is an error in the version definition of this chapter."
+translation_report:
+  heading: Αναφορά Μετάφρασης
+  original_title: Αρχικός Τίτλος
+  translation_title: Μεταφρασμένος Τίτλος
+  original_version: Αρχική Έκδοση
+  translation_version: Έκδοση Μετάφρασης
+  version_severity: Διαφορά Εκδόσεων
```

Comment:
You still need the English version, otherwise how people who dont know greek will translate these msgs 😀
Also, if the message in missin in a certain language it can default tonthe English one